toast-tk-runtime/src/main/java/io/toast/tk/runtime/mail/SmtpConfigProvider.java

```diff
@@ -0,0 +1,114 @@
+package io.toast.tk.runtime.mail;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.security.Security;
+import java.util.Properties;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import io.toast.tk.runtime.constant.Property;
+import io.toast.tk.runtime.utils.EncryptHelper;
+
+public class SmtpConfigProvider {
+
+	private static final Logger LOG = LogManager.getLogger(SmtpConfigProvider.class);
+
+	private String proxyURL;
+	private String proxyPort;
+	
+	private Properties properties;
+	
+	public static final String SMTP_PROPERTIES_FILE_NAME = "smtp.properties";
+	public static final String SMTP_PROPERTIES_FILE_STREAM_PATH = "/" + SMTP_PROPERTIES_FILE_NAME;
+	public static final String SMTP_PROPERTIES_FILE_PATH = Property.TOAST_HOME_DIR + SMTP_PROPERTIES_FILE_NAME;
+	
+	public static final String SMTP_PROPERTIES_HOST = "mail.smtp.host";
+
+	public static final String SMTP_PROPERTIES_PORT = "mail.smtp.port";
+
+	public static final String SMTP_PROPERTIES_TTLS_ENABLE = "mail.smtp.starttls.enable";
+
+	public static final String SMTP_PROPERTIES_AUTH = "mail.smtp.auth";
+
+	public static final String SMTP_PROPERTIES_USER = "mail.smtp.user";
+
+	public static final String SMTP_PROPERTIES_PASSWORD = "mail.smtp.password";
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Remove this hard-coded password. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2068)